### PR TITLE
yeats branch push

### PR DESCRIPTION
**Pull request by yeatsg**
Added header with my name.

### CODE REVIEW

**Review by MikeTroianello — APPROVED** [review]

**Review by Mihir9702 — COMMENTED** [review]
The future is now

**Review by gabedelgado — COMMENTED** [review]
noice

**Review by seeubanks12 — COMMENTED** [review]
Great work, yeats.

**Review by sergeykadiyevskiy — COMMENTED** [review]
Hello there!

File: index.html [hunk 1]
```diff
@@ -9,6 +9,7 @@
 </head>
 <body>
     <h3>Hi, my name is Mike</h3>
+    <h4>Yeats was here</h4>
```
Comment (NikoBads) [inline index.html:12]: Thanks yeats, very cool!

**Review by maiaduek — COMMENTED** [review]
*comment*

**Comment by dsilva06** [comment]
arepa

**Pull request merged**